.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+
+cache:
+  - pip
+
+matrix:
```

Comment:
Use python instead of matrix
See https://github.com/cupy/cupy/blob/master/.travis.yml#L7